examples/ffd/src/app_conf.h

```diff
@@ -107,6 +108,26 @@
 #define appconfI2S_MODE            appconfI2S_MODE_MASTER
 #endif
 
+#ifndef appconfI2C_MASTER_ENABLED
+#define appconfI2C_MASTER_ENABLED  1
+#endif
+
+#ifndef appconfI2C_SLAVE_ENABLED
+#define appconfI2C_SLAVE_ENABLED   0
+#endif
+
+#ifndef appconfI2C_SLAVE_DEVICE_ADDR
+#define appconfI2C_SLAVE_DEVICE_ADDR 0x42
+#endif
+
+#if appconfINTENT_I2C_OUTPUT_ENABLED && !appconfI2C_MASTER_ENABLED
```

Comment:
If `appconfINTENT_I2C_OUTPUT_ENABLED` means that the FFD example produces an intent over I2C, then it can be enabled when `appconfI2C_MASTER_ENABLED` is false.  The FFD example still produces intents.  It doesn't push them through an I2C master interface, but waits to be polled over an I2C slave interface.